### PR TITLE
move build to github action

### PR DESCRIPTION
**Pull request by jsdidierlaurent**

### CODE REVIEW

File: .github/workflows/continuous-integration.yml [hunk 1]
```diff
@@ -0,0 +1,177 @@
...
+    paths-ignore:
+      - 'docs/**'
+  pull_request:
+    paths-ignore:
+      - 'docs/**'
+
+# Jobs definition
+jobs:
+  test-core:
+    name: test - core
```
Comment (Alex-D) [inline .github/workflows/continuous-integration.yml]: "test / core"

File: .github/workflows/continuous-integration.yml [hunk 2]
```diff
@@ -0,0 +1,177 @@
...
+
+      - name: coverage
+        uses: codecov/codecov-action@v1.0.4
+        with:
+          token: ${{ secrets.codecov_token }}
+          yml: .github/.codecov.yml
+          flags: core
+
+  test-ui:
+    name: test - ui
```
Comment (Alex-D) [inline .github/workflows/continuous-integration.yml]: "test / ui"

File: test/pingdom.json [hunk 3]
```diff
@@ -0,0 +1,6 @@
+{
+  "columns": 6,
+  "tiles": [
+    { "type": "PINGDOM-CHECKS", "params": {}}
+  ]
+}
```
Comment (Alex-D) [inline test/pingdom.json]: Woot? GitHub Pingdom? :D

Comment (jsdidierlaurent) [reply test/pingdom.json]: Oups ^^

**Pull request merged**